infra/azure.tf

```diff
@@ -145,10 +145,31 @@ CRON
 
 chmod +x /root/daily-reset.sh
 
+cat <<GET_TARGETS > /root/get-targets.sh
```

Comment:
```suggestion
cat <<'GET_TARGETS' > /root/get-targets.sh
```

Reply:
@garyverhaegen-da, I'm sorry, I don't know what happened but yesterday I thought I had committed the `suggestion` above but clearly that wasn't the case. Since I couldn't find any PR from yours touching this file, I've opened https://github.com/digital-asset/daml/pull/18116